lib/liquid/standardfilters.rb

```diff
@@ -394,6 +393,12 @@ def default(input, default_value = ''.freeze)
 
     private
 
+    def safe_property_check(property)
+      yield
+    rescue TypeError
+      raise Liquid::ArgumentError.new("Cannot select the property '#{property}'")
```

Comment:
nit: we interpolate errors as `Liquid error: <error message>`, and this reads more nicely with a lowercase error message.

Reply:
👍 ^